src/common/proc_args.c

```diff
@@ -598,6 +603,8 @@ static long _str_to_mbtyes(const char *arg, int use_gbytes)
 	result = strtol(arg, &endptr, 10);
```

Comment:
Nope, if we're going to go the full 64 bit we need something else, see https://stackoverflow.com/a/17003732/349809.